[2025-06-05T18:31:56.561+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: deep_models_pipeline.train_deep_user_model manual__2025-06-05T18:31:52.266088+00:00 [queued]>
[2025-06-05T18:31:56.569+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: deep_models_pipeline.train_deep_user_model manual__2025-06-05T18:31:52.266088+00:00 [queued]>
[2025-06-05T18:31:56.569+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 3
[2025-06-05T18:31:56.579+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): train_deep_user_model> on 2025-06-05 18:31:52.266088+00:00
[2025-06-05T18:31:56.585+0000] {standard_task_runner.py:60} INFO - Started process 1742 to run task
[2025-06-05T18:31:56.587+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'deep_models_pipeline', 'train_deep_user_model', 'manual__2025-06-05T18:31:52.266088+00:00', '--job-id', '1972', '--raw', '--subdir', 'DAGS_FOLDER/train_deep_model_dag.py', '--cfg-path', '/tmp/tmpnqjqeotj']
[2025-06-05T18:31:56.588+0000] {standard_task_runner.py:88} INFO - Job 1972: Subtask train_deep_user_model
[2025-06-05T18:31:56.618+0000] {task_command.py:423} INFO - Running <TaskInstance: deep_models_pipeline.train_deep_user_model manual__2025-06-05T18:31:52.266088+00:00 [running]> on host 380fc082ee8f
[2025-06-05T18:31:56.672+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='deep_models_pipeline' AIRFLOW_CTX_TASK_ID='train_deep_user_model' AIRFLOW_CTX_EXECUTION_DATE='2025-06-05T18:31:52.266088+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-05T18:31:52.266088+00:00'
[2025-06-05T18:31:56.673+0000] {train_deep_model_dag.py:13} INFO - üü¶ Running command: python /opt/***/src/models/train_user_deep_model.py --n_neighbors=10 --latent_dim=32
[2025-06-05T18:37:12.279+0000] {train_deep_model_dag.py:20} INFO - üü© [stdout]:
üèÉ View run train_user_deep_model at: https://dagshub.com/sacer11/MLOps_movie_rec_apr25.mlflow/#/experiments/6/runs/9e70422d90db4d37baa0d1afa80e342f
üß™ View experiment at: https://dagshub.com/sacer11/MLOps_movie_rec_apr25.mlflow/#/experiments/6

[2025-06-05T18:37:12.280+0000] {train_deep_model_dag.py:22} WARNING - üü® [stderr]:
/home/***/.local/lib/python3.10/site-packages/mlflow/pyfunc/utils/data_validation.py:186: UserWarning: [33mAdd type hints to the `predict` method to enable data validation and automatic signature inference during model logging. Check https://mlflow.org/docs/latest/model/python_model.html#type-hint-usage-in-pythonmodel for more details.[0m
  color_warning(
2025-06-05 18:35:07,062 - __main__ - INFO - Epoch 1/25: Loss = 0.0005
2025-06-05 18:35:09,578 - __main__ - INFO - Epoch 2/25: Loss = 0.0000
2025-06-05 18:35:11,942 - __main__ - INFO - Epoch 3/25: Loss = 0.0000
2025-06-05 18:35:14,449 - __main__ - INFO - Epoch 4/25: Loss = 0.0000
2025-06-05 18:35:16,874 - __main__ - INFO - Epoch 5/25: Loss = 0.0000
2025-06-05 18:35:19,268 - __main__ - INFO - Epoch 6/25: Loss = 0.0000
2025-06-05 18:35:21,716 - __main__ - INFO - Epoch 7/25: Loss = 0.0000
2025-06-05 18:35:24,159 - __main__ - INFO - Epoch 8/25: Loss = 0.0000
2025-06-05 18:35:26,559 - __main__ - INFO - Epoch 9/25: Loss = 0.0000
2025-06-05 18:35:29,035 - __main__ - INFO - Epoch 10/25: Loss = 0.0000
2025-06-05 18:35:31,429 - __main__ - INFO - Epoch 11/25: Loss = 0.0000
2025-06-05 18:35:33,835 - __main__ - INFO - Epoch 12/25: Loss = 0.0000
2025-06-05 18:35:36,229 - __main__ - INFO - Epoch 13/25: Loss = 0.0000
2025-06-05 18:35:38,574 - __main__ - INFO - Epoch 14/25: Loss = 0.0000
2025-06-05 18:35:41,024 - __main__ - INFO - Epoch 15/25: Loss = 0.0000
2025-06-05 18:35:43,421 - __main__ - INFO - Epoch 16/25: Loss = 0.0000
2025-06-05 18:35:45,886 - __main__ - INFO - Epoch 17/25: Loss = 0.0000
2025-06-05 18:35:48,311 - __main__ - INFO - Epoch 18/25: Loss = 0.0000
2025-06-05 18:35:50,667 - __main__ - INFO - Epoch 19/25: Loss = 0.0000
2025-06-05 18:35:53,140 - __main__ - INFO - Epoch 20/25: Loss = 0.0000
2025-06-05 18:35:55,576 - __main__ - INFO - Epoch 21/25: Loss = 0.0000
2025-06-05 18:35:57,882 - __main__ - INFO - Epoch 22/25: Loss = 0.0000
2025-06-05 18:36:00,342 - __main__ - INFO - Epoch 23/25: Loss = 0.0000
2025-06-05 18:36:02,774 - __main__ - INFO - Epoch 24/25: Loss = 0.0000
2025-06-05 18:36:05,171 - __main__ - INFO - Epoch 25/25: Loss = 0.0000
2025-06-05 18:36:08,294 - __main__ - INFO - ‚úÖ User-Embeddings gespeichert unter /opt/***/data/processed/user_deep_embedding.csv
2025-06-05 18:36:08,904 - __main__ - INFO - ‚úÖ Deep KNN Modell gespeichert unter /opt/***/models/user_deep_knn.pkl
2025-06-05 18:36:08,906 - __main__ - INFO - ‚úÖ Parameter als JSON gespeichert: /opt/***/models/user_deep_model_params.json
2025/06/05 18:36:38 INFO mlflow.pyfunc: Validating input example against model signature

Downloading artifacts:   0%|          | 0/1 [00:00<?, ?it/s]
Downloading artifacts: 100%|‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà| 1/1 [00:00<00:00,  2.37it/s]
Downloading artifacts: 100%|‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà| 1/1 [00:00<00:00,  2.37it/s]
Downloading artifacts: 100%|‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà| 1/1 [00:00<00:00,  2.37it/s]
Registered model 'user_deep_model' already exists. Creating a new version of this model...
2025/06/05 18:37:09 INFO mlflow.store.model_registry.abstract_store: Waiting up to 300 seconds for model version to finish creation. Model name: user_deep_model, version 8
Created version '8' of model 'user_deep_model'.
2025-06-05 18:37:10,369 - __main__ - INFO - üìù Tags f√ºr Modellversion 8 gesetzt: n_neighbors=10, latent_dim=32
2025-06-05 18:37:10,733 - __main__ - INFO - üèÅ Deep User-Model Training abgeschlossen und geloggt.

[2025-06-05T18:37:12.280+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-05T18:37:12.289+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=deep_models_pipeline, task_id=train_deep_user_model, execution_date=20250605T183152, start_date=20250605T183156, end_date=20250605T183712
[2025-06-05T18:37:12.310+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-05T18:37:12.329+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
